test/cmdlineTests/standard_wrong_key_auxiliary_input.json

```diff
@@ -1,14 +0,0 @@
-{
-	"language": "Solidity",
-	"sources":
-	{
-		"A":
-		{
-			"content": "pragma solidity >=0.0; contract C { function f() public pure {} }"
-		}
-	},
-	"auxiliaryInput":
-	{
-		"key1": "test"
```

Comment:
It's a bit hard to see, but I think this one was replaced by a copy of ``standard_wrong_type_auxiliary_input`` by accident. I think it needs to be restored.